MetabolicAtlas/MetabolicAtlas
# feat: gotenzymes formatted formulas

**Related issue(s) and PR(s)**  
This PR closes #1113.



**Type of change**  


 
**List of changes made**  

- Formulas for compounds are formatted (using existing code)
- Equations for reactions are formatted (new code)

**Screenshot of the fix**  

![2022-09-19-162741_845x173_scrot](https://user-images.githubusercontent.com/1029190/191047799-b3fc5e09-033e-475f-a9b7-4421b77790b1.png)

![2022-09-19-165015_419x172_scrot](https://user-images.githubusercontent.com/1029190/191046517-025d036a-78b6-423d-9b69-21c7fadcefe8.png)


**Testing**  

- Inspect the formula at `gotenzymes/compound/C00097`
- Inspect the equation at `gotenzymes/reaction/R01677`
- Inspect the new function [`chemicalEquation`](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/4e94d7bfd02a3c932522e52e6f2fe41cf7871e44/frontend/src/helpers/chemical-formatters.js#L22)
- Please browse different details pages under GotEnzymes to make sure everything looks ok.

**Further comments**
- No tests are added since this is frontend code, so the acceptance criteria are not fulfilled.
- Do we want links from the equation sections, as for the GEM browser?
![2022-09-19-162841_821x160_scrot](https://user-images.githubusercontent.com/1029190/191048042-c4fa8a56-5e86-4cba-8483-ec613e9cba9b.png)


**Definition of Done checklist**
